test/libsolidity/SemanticTest.cpp

```diff
@@ -197,6 +245,21 @@ TestCase::TestResult SemanticTest::runTest(ostream& _stream, string const& _line
 			bytes output;
 			if (test.call().kind == FunctionCall::Kind::LowLevel)
 				output = callLowLevel(test.call().arguments.rawBytes(), test.call().value.value);
+			else if (test.call().kind == FunctionCall::Kind::Builtin)
+			{
+				std::vector<string> builtinPath;
+				boost::split(builtinPath, test.call().signature, boost::is_any_of("."));
+				soltestAssert(builtinPath.size() == 2, "");
+				auto builtin = m_builtins[builtinPath.front()][builtinPath.back()];
+				std::optional<bytes> builtinOutput{builtin->builtin(test.call())};
```

Comment:
```suggestion
				optional<bytes> builtinOutput{builtin->builtin(test.call())};
```

Reply:
Also `std::optional` gets ambiguous because of `boost::optional`.